routing/router.go

```diff
@@ -1653,131 +1656,124 @@ func (r *ChannelRouter) sendPayment(payment *LightningPayment,
 			return [32]byte{}, nil, err
 		}
 
-		// Send payment attempt. It will return a final boolean
-		// indicating if more attempts are needed.
-		preimage, final, err := r.sendPaymentAttempt(
-			paySession, route, payment.PaymentHash,
+		log.Tracef("Attempting to send payment %x, using route: %v",
+			paymentHash, newLogClosure(func() string {
+				return spew.Sdump(route)
```

Comment:
do we need to nil the curve here?

Reply:
Maybe a separate PR, since this only moves code? :)